hvplot/converter.py

```diff
@@ -1043,7 +1065,7 @@ def __init__(
         if kind == 'errorbars':
             hover = False
         elif hover is None:
-            hover = not self.datashade
+            hover = not self.datashade if not self.selector else True
```

Comment:
```suggestion
            hover = True if self.selector else not self.datashade
```

Easier for me to read.